packages/wonder-blocks-banner/src/components/banner.js

```diff
@@ -196,10 +203,12 @@ const Banner = (props: Props): React.Node => {
                 ]}
             />
             <View style={styles.containerInner}>
-                <img
-                    src={iconForKind(kind ?? "info")}
-                    alt={kind}
-                    className={css(styles.icon)}
+                <Icon
+                    icon={iconForKind(kind ?? "info")}
```

Comment:
If you align the names of the exports from banner-icons.js with `BannerKind`, you could drop the `iconForKind` banner.  It would look something like:
```
import * as bannerIcons from "./banner-icons.js";

...

<Icon icon={bannerIcons[kind]} ... />
```